src/database/mod.rs

```diff
@@ -1,17 +1,15 @@
-use rocket_contrib::databases::diesel;
-
 pub mod articles;
 pub mod comments;
 pub mod profiles;
 pub mod users;
 
 #[database("diesel_postgres_pool")]
-pub struct Conn(diesel::PgConnection);
+pub struct Db(diesel::PgConnection);
```

Comment:
https://github.com/SergioBenitez/Rocket/blob/master/examples/databases/src/diesel_sqlite.rs

Reply:
I changed the naming based off one of the official examples. I figured people would probably be referring back to that at some point.